wurst/systems/core/PlayerConfig.wurst

```diff
@@ -0,0 +1,35 @@
+package PlayerConfig
+
+import HashMap
+import GameMode
+
+public class PlayerConfig
+    static let playerToConfigMap = new HashMap<player, PlayerConfig>()
```

Comment:
This should use `thistype` in the generic.

Reply:
I don't understand, do you mean this? This doesn't work cannot infer type for key/value.
```
public class PlayerConfig
    static let playerToConfigMap = new HashMap<player, this>()
```

Reply:
Nevermind, it should be as you wrote `thistype `not `this`, my bad.